pyomo/common/errors.py

```diff
@@ -25,3 +25,5 @@ def __str__(self):
                  % ( repr(self.parameter), ) )
 
 
+class InfeasibleConstraintException(Exception):
```

Comment:
Should we have documentation to the effect of "This exception should be raised by Pyomo transformations to indicate that an infeasible constraint has been identified."?